frontend/src/metabase/admin/upsells/components/UpsellBigCard.tsx

```diff
@@ -0,0 +1,74 @@
+import { useMount } from "react-use";
+
+import ExternalLink from "metabase/core/components/ExternalLink";
+import { Box, Flex, Image, Stack, Text, Title } from "metabase/ui";
+
+import { UpsellGem } from "./UpsellGem";
+import { UpsellWrapper } from "./UpsellWrapper";
+import S from "./Upsells.module.css";
+import { trackUpsellClicked, trackUpsellViewed } from "./analytics";
+import { useUpsellLink } from "./use-upsell-link";
+
+export type UpsellBigCardProps = {
+  title: string;
+  buttonText: string;
+  buttonLink: string;
+  campaign: string;
+  source: string;
+  illustrationSrc?: string;
+  children: React.ReactNode;
```

Comment:
Certainly not blocking, but I prefer to use the `React.PropsWithChildren<>` type when props have children.

Reply:
Gotcha.
I'll update it in the next PR (that still touches this component).

Reply:
https://linear.app/metabase/issue/CLO-3576/use-reactpropswithchildren